### PR TITLE
run hammer ping on Foreman too

### PR DESCRIPTION
**Pull request by evgeni**
this exists since 1.24

### CODE REVIEW

File: bats/fb-test-foreman.bats [hunk 1]
```diff
@@ -13,6 +16,29 @@ set -o pipefail
   [ `curl -sk "https://localhost$URL_PREFIX/status" | ruby -e "require 'json'; puts JSON.load(ARGF.read)['status']"` == 'ok' ]
 }
 
+@test "check hammer ping" {
+  FOREMAN_VERSION=$(tForemanVersion)
+  if [ $FOREMAN_VERSION == '1.24' ]; then
+    # https://github.com/theforeman/hammer-cli-foreman/pull/394
+    # https://github.com/Katello/hammer-cli-katello/pull/596
+    skip "Functionality is broken in 1.24"
```
Comment (evgeni) [inline bats/fb-test-foreman.bats:24]: I kept this skip, as Satellite 6.7 is still supported, and is based on 1.24.

File: bats/fb-test-foreman.bats [hunk 2]
```diff
@@ -13,6 +16,29 @@ set -o pipefail
...
+  fi
+
+  local next_wait_time=0
+  until [ "${status:-1}" -eq 0 -o $next_wait_time -eq 12 ]; do
+    run hammer --reload-cache ping
```
Comment (evgeni) [inline bats/fb-test-foreman.bats:29]: and I've added `--reload-cache` here, so that the apidoc is always fresh

File: bats/fb-test-katello.bats [hunk 3]
```diff
@@ -6,29 +6,6 @@ set -o pipefail
...
-  echo "${output}"
-
-  [ $status -eq 0 ]
-
-  # Hammer exits with 0 on failures
-  # https://projects.theforeman.org/issues/30496
-  [[ $output != *"FAIL"* ]]
-}
-
 @test "check service status" {
```
Comment (ekohl) [inline bats/fb-test-katello.bats:9]: Should this be `fb-test-foreman-maintain.bats` now?

**Review by ekohl — APPROVED** [review]
I thought I submitted a PR about this but clearly not. However, I had the exact same thought in the past.

File: bats/fb-test-katello.bats (hunk 3 above)
Comment (evgeni) [reply bats/fb-test-katello.bats:9]: In theory, yes. Or maybe even foreman.bats with a `if package installed` guard. But I'd like to leave that as an exercise to the reader.

**Pull request merged**